flink-kubernetes-operator/src/test/resources/log4j2-test.properties

```diff
@@ -34,7 +34,7 @@
 # limitations under the License.
 ################################################################################
 
-rootLogger.level = OFF
+rootLogger.level = INFO
```

Comment:
Using `INFO` log level makes debugging failed tests easier. Hope it does not have other side effects.